frontend/lib/framework/resolution-error-classes-list.cpp

```diff
@@ -336,6 +336,33 @@ void ErrorMultipleQuestionArgs::write(ErrorWriterBase& wr) const {
   wr.code(secondQuestion, { secondQuestion });
 }
 
+void ErrorNestedClassFieldRef::write(ErrorWriterBase& wr) const {
+  auto outerDecl = std::get<0>(info);
+  auto innerDecl = std::get<1>(info);
+  auto reference = std::get<2>(info);
+  auto id = std::get<3>(info);
+
+  const char* outerName = outerDecl->isClass() ? "class" : "record";
```

Comment:
Don't forget `union` here, lol. Or maybe just assert it's not possible, idk. I wouldn't seriously worry about unions because I doubt anyone uses them.

Reply:
Added an assertion; my only thought for how you could trigger this with unions was an inner union using an outer record's type field, but that's apparently not possible.